### DISCUSSION TITLE
how can we remove uwebsockets version from headers?

### DISCUSSION THREAD

**Discussion by dhruv-nudge** [uNetworking/uWebSockets.js#1070] [General]
we are always sending uwebsockets version header, can we remove it?

**Comment by e3dio** [comment]
Edit source code or have proxy remove it or there is config option but I think you need to register with uNetAb

**Reply by dhruv-nudge** [reply]
how does the registration with nNetAb work?

**Reply by uNetworkingAB** [reply]
Are you making a proprietary app or something open source?

**Reply by dhruv-nudge** [reply]
proprietary app

**Reply by uNetworkingAB** [reply]
Ok send a mail

**Comment by joshxyzhimself** [comment]
doable in reverse proxies like caddy, haproxy, traefik, etc.

but that's like removing the logo at the front hood of your koenigsegg sports car.
